src/test/ui/macros/issue-98466.stderr

```diff
@@ -2,80 +2,80 @@ warning: named argument `_x` is not used by name
   --> $DIR/issue-98466.rs:7:26
    |
 LL |     println!("_x is {}", _x = 5);
-   |                     --   ^^ this named argument is only referred to by position in formatting string
-   |                     |
-   |                     this formatting argument uses named argument `_x` by position
+   |                      -   ^^ this named argument is only referred to by position in formatting string
```

Comment:
is the span change only underlining the closing `}` expected ?

Reply:
@lqd
Basically yes, because that's where the named argument gets inserted. But now that you've commented on it, I think it should probably underline both.

I think I will have time to fix this today.

Reply:
Took a little longer than I expected!
https://github.com/rust-lang/rust/pull/99958